doc/release-notes.md

```diff
@@ -1,11 +1,14 @@
-Dash Core version 0.15
+Dash Core version 0.16
 ======================
 
 Release is now available from:
 
   <https://www.dash.org/downloads/#wallets>
 
-This is a new major version release, bringing new features, various bugfixes and other improvements.
+This is a new major version release, bringing new features, various bugfixes
+and other improvements.
```

Comment:
```suggestion
and other improvements.

This version is the result of around 7 months of hard work consisting of more than 270 pull requests and totaling more than 900 commits.
```

Reply:
These are pretty vague metrics imo (not much better than counting deleted/added lines), would rather avoid that.

Reply:
Okay, just a thought